web/httpserver.py

```diff
@@ -35,7 +36,7 @@ def runbasic(func, server_address=("0.0.0.0", 8080)):
 
     class WSGIHandler(SimpleHTTPRequestHandler):
         def run_wsgi_app(self):
-            protocol, host, path, parameters, query, fragment = urlparse.urlparse(
+            protocol, host, path, parameters, query, fragment = urlparse(
```

Comment:
This one is not quite right.  The imports need to be modified to match `web/db.py`

Reply:
Sorry, I forget this for a long time, updated!